Doc/whatsnew/3.11.rst

```diff
@@ -266,6 +267,9 @@ math
   ``inf``. Previously they raised :exc:`ValueError`. (Contributed by Mark
   Dickinson in :issue:`44339`.)
 
+* The :data:`math.nan` value is now available available.
```

Comment:
"available" duplicated, done on purpose?

Reply:
Same in the PR title :)